apps/studio/src/components/CoreTabs.vue

```diff
@@ -344,6 +345,7 @@ export default Vue.extend({
     },
     keymap() {
       const result = {
+        'ctrl+shift+T': this.reopenLastClosedTab,
```

Comment:
I think this is probably better to store this value in the TabModule of the vue store -- that way we don't need to remember to set `lastClosedTab` anywhere we close a tab.

Some other things we'll need to do here:
- We'll want to create a new `OpenTab` with a clone of the values in the specific tab because this is an ORM model and will include an `id`, but when `$store.dispatch('tabs/remove', tab)` is called we delete it from the database. So we'll need to add an optional constructor argument to copy values from.
- We should store array of `recently closed tabs` in the store, that way calling `reopenClosedTab` can be clicked multiple times until the stack runs out!

With this new setup, to restore a tab we could just call `this.$store.dispatch('reopenTab')` and it will re-add the tab most recently closed

Reply:
Hi @rathboma 
thanks for the review! yeah that all make sense.. I'll take a look 😄